presto-main/src/test/java/io/prestosql/operator/TestHashAggregationOperator.java

```diff
@@ -196,20 +198,22 @@ public void testHashAggregation(boolean hashEnabled, boolean spillEnabled, long
 
         DriverContext driverContext = createDriverContext(memoryLimitForMerge);
 
-        MaterializedResult expected = resultBuilder(driverContext.getSession(), VARCHAR, BIGINT, BIGINT, DOUBLE, VARCHAR, BIGINT, BIGINT)
-                .row("0", 3L, 0L, 0.0, "300", 3L, 3L)
-                .row("1", 3L, 3L, 1.0, "301", 3L, 3L)
-                .row("2", 3L, 6L, 2.0, "302", 3L, 3L)
-                .row("3", 3L, 9L, 3.0, "303", 3L, 3L)
-                .row("4", 3L, 12L, 4.0, "304", 3L, 3L)
-                .row("5", 3L, 15L, 5.0, "305", 3L, 3L)
-                .row("6", 3L, 18L, 6.0, "306", 3L, 3L)
-                .row("7", 3L, 21L, 7.0, "307", 3L, 3L)
-                .row("8", 3L, 24L, 8.0, "308", 3L, 3L)
-                .row("9", 3L, 27L, 9.0, "309", 3L, 3L)
-                .build();
+        MaterializedResult.Builder expectedBuilder = resultBuilder(driverContext.getSession(), VARCHAR, BIGINT, BIGINT, DOUBLE, VARCHAR, BIGINT, BIGINT);
+        for (int i = 0; i < numberOfRows; ++i) {
+            expectedBuilder.row(Integer.toString(i), 3L, 3L * i, (double) i, Integer.toString(300000 + i), 3L, 3L);
+        }
+
+        List<Page> pages = toPages(operatorFactory, driverContext, input);
+        assertGreaterThan(pages.size(), 0, "Expected more then one output page");
+        if (hashEnabled) {
+            // Drop the hashChannel for all pages for comparison
+            pages = dropChannel(pages, hashChannels);
+        }
+
+        MaterializedResult expected = expectedBuilder.build();
+        MaterializedResult actual = toMaterializedResult(driverContext.getSession(), expected.getTypes(), pages);
+        assertEqualsIgnoreOrder(actual.getMaterializedRows(), expected.getMaterializedRows());
```

Comment:
I don't get it. Previously the code was calling `assertOperatorEqualsIgnoreOrder`
Now it's more verbose, including manually calling `toPages`, dropping hash channels, converting to `MaterializedResult`.
i assume this is `assertOperatorEqualsIgnoreOrder` inlined, but what's the actual difference?

Is this really related to increasing the data set, as `Produce more than single page in testHashAggregation` commit message says?